src/theory/bv/theory_bv_rewrite_rules_simplification.h

```diff
@@ -21,6 +21,7 @@
 
 #include "theory/bv/theory_bv_rewrite_rules.h"
 #include "theory/bv/theory_bv_utils.h"
+#include "theory/rewriter.h"
```

Comment:
Mostly curious (and a bit paranoid), is there a cycle between these headers?

Reply:
Nope, no cycles here. I added the header since `Rewriter::rewrite` is used.